contribs/src/main/java/com/netflix/conductor/contribs/publisher/TaskStatusPublisher.java

```diff
@@ -54,7 +57,7 @@ public void run() {
             this.setUncaughtExceptionHandler(new ExceptionHandler());
             String tName = Thread.currentThread().getName();
             LOGGER.info("{}: Starting consumer thread", tName);
-            Task task;
+            Task task = null;
```

Comment:
Is this required ? i .e  assigning null.

Reply:
yes as its used in the catch block

Reply:
otherwise its reporting that variable may not be initialized.